core/src/main/scala/kafka/server/ReplicaAlterLogDirsThread.scala

```
@@ -159,22 +161,39 @@ class ReplicaAlterLogDirsThread(name: String,
     }
   }
 
-  override protected def fetchEarliestOffsetFromLeader(topicPartition: TopicPartition, leaderEpoch: Int): Long = {
+  override protected def fetchEarliestLocalOffsetFromLeader(topicPartition: TopicPartition, leaderEpoch: Int): (Int, Long) = {
```

This is not supported yet, right? Should we throw an exception?